MANIFEST.in

```diff
@@ -1 +1,11 @@
-include AUTHORS COPYING README.md
```

Comment:
Right missed that, according to http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files I can use 'exclude' directive. Will change.